python/ray/autoscaler/aws/cloudwatch/prometheus_ray_up.sh

```diff
@@ -0,0 +1,42 @@
+###########################################################################################################
+#This bash script is meant to support prometheus metrics collection via cloudwatch agent.
+#You can view the ray metrics being collected through AWS CloudWatch console.
+###########################################################################################################
+
+USAGE_INSTRUCTIONS='Usage: prometheus_ray_up.sh [OPTIONS] CLUSTER_CONFIG_FILE'
+
+# ray up to start a new cluster
+ray up "$@"
+
+if [ $? -eq 0 ]; then
+  AUTOSCALER_CONFIG_FILE=$(sed 's/.* //' <<< "$@")
+  if [ ! -f "$AUTOSCALER_CONFIG_FILE" ]; then
+    echo "Cluster config file not found"
```

Comment:
Minor: To simplify debugging, I'd recommend also printing the autoscaler config file path that wasn't found (e.g. echo "Cluster config file not found: $AUTOSCALER_CONFIG_FILE").